app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -437,6 +438,13 @@ object ComponentRegistry {
         TAB_TITLE_VIEW story NodeData { Story of R.layout.component_list },
       )
     ),
+    "Skeleton" story NodeData(
```

Comment:
same here - could you move this into the right place for alphabetical order

Reply:
ok